apps/dash-lunar-lander/requirements.txt

```diff
@@ -0,0 +1,4 @@
+dash
+casadi
```

Comment:
Specify the version of the packages you're using, this could avoid the occasion that an updated version of package leads to your app fail due to reasons like API deprecation.

```
dash==1.8.0
```